src/api/app/views/webui2/webui/project/_pulse_list.html.haml

```diff
@@ -0,0 +1,91 @@
+.row
+  .col
+    %p
+      During this period
+      = render partial: 'pulse_list_commits'
+    %p
+      = render partial: 'pulse_list_builds'
+      = render partial: 'pulse_list_new'
+      = render partial: 'pulse_list_changes'
+    %p
+      = render partial: 'pulse_list_branches'
+      = render partial: 'pulse_list_comments'
+
+  .col
+    .card
+      .card-header
+        Requests
+      .card-body
+        - if @requests.any?
+          .row
+            .col
+              .progress
+                -# haml-lint:disable InlineStyles
+                - @requests_by_percentage.each do |state, percentage|
+                  %div{ title: "#{@requests_by_state[state]} #{state} requests",
+                        class: "progress-bar progress-state-#{state}",
+                        'aria-valuemax': '100', 'aria-valuemin': '0', 'aria-valuenow': percentage,
+                        role: 'progressbar', style: "width: #{percentage}%" }
+                -# haml-lint:enable InlineStyles
+              %p
+                = link_to('#pulse-requests') do
+                  = @requests_by_state.values.sum
+                  active requests
+          .row
+            - @requests_by_state.each_key do |state|
+              .col.border.text-center
+                %p.fa-3x
+                  %i{ class: "fa #{request_bootstrap_icon(state)} request-state-#{state}" }
```

Comment:
%i.fa{ class: "#{request_bootstrap_icon(state)} request-state-#{state}" }

Reply:
I think what @dmarcoux says should be changed

Reply:
Because?